plotter/filledLine_test.go

```diff
@@ -16,7 +16,7 @@ import (
 	"gonum.org/v1/plot/plotter"
 )
 
-func ExampleLine_filledLine() {
+func clippedFilledLine() {
```

Comment:
Add comment linking to #488. `// See https://github.com/gonum/plot/issues/488`.

Reply:
Done